WTWSMS/common/buildings/WtWSMS_historical_temple.txt

```diff
@@ -0,0 +1,183 @@
+temple = {
+
+	tp_hagiasophia = {  # Hagia Sophia
+		potential = {
+			FROMFROM = { title = b_hagiasophia }
+			FROM = { had_global_flag = { flag = hagia_sophia_built days = 0 } }
+		}
```

Comment:
I used the had_global_flag to hide the building, when I last tested it couldn't be built directly. I will check again.
`always = no` should be used in potential or trigger?